persistence/test/account_test.go

```diff
@@ -294,13 +303,18 @@ func newTestAccount(t *testing.T) typesGenesis.Account {
 	}
 }
 
+func createAndInsertNewPool(db *persistence.PostgresContext) (*genesis.Pool, error) {
+	pool := newTestPool(nil)
+	return &pool, db.SetPoolAmount(pool.Name, DefaultAccountAmount)
+}
+
 func newTestPool(t *testing.T) typesGenesis.Pool {
 	_, err := crypto.GenerateAddress()
 	if t != nil {
 		require.NoError(t, err)
 	}
 	return typesGenesis.Pool{
-		Name: DefaultPoolName,
+		Name: fmt.Sprintf("%s_%d", DefaultPoolName, rand.Int()),
```

Comment:
(Non-Actionable) Pools become proto-enum in #154

Reply:
ACK